### PR TITLE
Chore: kill, Kill, KILL the selenium process with tree-kill.

### PR DESCRIPTION
**Pull request by ryan-roemer**
Uses additional `tree-kill` with `SIGKILL` to nuke Selenium.

/cc @coopy @kkerr1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
   "dependencies": {
     "colors": "1.1.0",
     "lodash": "3.8.0",
-    "portscanner": "^1.0.0"
+    "portscanner": "^1.0.0",
```
Comment (coopy) [inline package.json]: Do we need portscanner? Is it a peerDep of `treeKill`?

Comment (ryan-roemer) [inline package.json]: haha, no worries

File: lib/server.js [hunk 2]
```diff
@@ -207,8 +208,12 @@ Server.prototype.stop = function (callback) {
...
     callback(self._err);
   });
-
   proc.on("exit", _done);
+
+  // Kill it the "official" way.
   proc.kill();
+
+  // Kill it the unofficial way.
+  treeKill(proc.pid, "SIGKILL", _done);
```
Comment (coopy) [inline lib/server.js]: Are there any consequences of sending such a strong kill signal? Will the process clean up everything synchronously thru the regular `proc.kill()`?

**Comment by coopy** [comment]
LGTM if the `SIGKILL` isn't a problem.

File: lib/server.js (hunk 2 above)
Comment (ryan-roemer) [inline lib/server.js]: So, this utility actually shells to command line (linux, mac, windows, etc) and sends the proper signal. There's only one process coming back from `selenium-standalone`, so should be safe.

**Comment by coopy** [comment]
model rocket burn

**Comment by ryan-roemer** [comment]
Released as `rowdy@0.3.6`

**Pull request merged**